library/clefos

```diff
@@ -2,7 +2,7 @@ Maintainers: The ClefOS Project <neale@sinenomine.net> (@nealef)
 GitRepo: https://github.com/nealef/clefos.git
 Constraints: !aufs
 
-Tags: 7, 7.4.1708, latest
-GitFetch: refs/heads/7.4.1708
-GitCommit: 002827d6daa02a1c26d53c2571cc9024968f21d2
+Tags: 7, 7.5.1804, latest
+GitFetch: refs/heads/7.5.1804
+GitCommit: a13384382290c7aef7012d1353fee4f84ce8548c
```

Comment:
Commit value looks fine, but `GitFetch` should still point to the branch, which is still `7.4.1708`. :+1:

Reply:
Only just saw this reply. I am not sure what you mean. I should have two entries: 1 for 7.5 and the other (existing) for 7.4?

Reply:
No, I mean `GitFetch:` is referring to your Git branch name -- in your case, that's still 7.4.1708 (https://github.com/nealef/clefos/tree/7.4.1708).